content/partials/general/events/_batched_events.textile

```diff
@@ -1,8 +1,8 @@
 Each batched message will have the following fields:
 
-- name := the event type, aka "@presence.message@", "@channel.message@", "@channel.closed@", etc
+- name := the event type, for example @presence.message@, @channel.message@ or @channel.closed@
 - webhookId := an internal unique ID for the configured webhook
-- source := the source for the webhook, namely "@channel.message@", "@channel.presence@", "@channel.lifecycle@"
+- source := the source for the webhook, which will be @channel.message@, @channel.presence@ or @channel.lifecycle@
 - timestamp := a timestamp represented as milliseconds since the epoch for the presence event
```

Comment:
Looks good. I just wondered if it would be better to use "Unix timestamp" here instead of "milliseconds since the epoch"?

Reply:
I definitely agree we need to move away from "milliseconds since the epoch". My thinking is that we should address these in bulk though, as we have about ~150 occurrences of it in the docs. I'll raise a JIRA to replace them and leave this for now so we're consistently "wrong", if you agree?

Reply:
Sounds like a good plan to me @m-hulbert !

Reply:
Great! Created that and assigned it to me.

Reply:
(fwiw I'm used to "Unix timestamp" with no further qualifications meaning seconds since the epoch, not milliseconds. "Unix timestamp in milliseconds" might work)

Reply:
Thanks @SimonWoolf - that's a good point and I'll make a note of it in the issue I raised.